### DISCUSSION TITLE
Integration Dashboards aren't showing data

### DISCUSSION THREAD

**Discussion by Larockus** [Security-Onion-Solutions/securityonion#11930] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

12

### RAM

30Gig

### Storage for /

2Tb

### Storage for /nsm

212G

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have my Meraki appliance and O365 sending logs into my installation, but none of the integration specific dashboards show any data. I can search for the specific types of logs but I'm really wanting to have the dashboards available if possible. 

O365 dashboard isn't showing anything and the Meraki is throwing the errors displayed below. I'm not sure where to look from here. 

![Screenshot 2023-12-04 111228](https://github.com/Security-Onion-Solutions/securityonion/assets/108535113/7165b52c-1f76-4998-aa40-a09343db8a35)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by lancebritton** [comment]
I am having the same issue with the cisco_asa integration.

**Comment by defensivedepth** [comment]
@lancebritton @Larockus Can you confirm that you are getting logs for the integration in the Security Onion Console - Hunt or Dashboards?

**Reply by lancebritton** [reply]
I am not getting logs visible in the Security Onion Console - Hunt or Dashboards.

**Reply by defensivedepth** [reply]
Ok that would be why the Integration dashboards aren't working - because there is no data. 

Please create a new Discussion and make sure to fill out the technical questions so that we can help you more easily.

**Reply by Larockus** [reply]
I'm not seeing either the Meraki or the 0365 logs in the SOC dashboard/Hunt. So based off your last comment to the other poster there is no data. How can I determine where the hang up in the collection is? I looked through the logs earlier before posting, but I didn't see anything of merit. Does 2.4 have a method to increase logging to debug like 2.3?

**Reply by defensivedepth** [reply]
The Integrations are attached to a Policy, and then the policy is assigned to an Agent - that agent's logs is where you will need to start looking. You can do that from the Fleet interface in Kibana - click on the Agent, then the Log's tab. You can also change the default logging level of a particular agent in that screen as well.

**Reply by defensivedepth** [reply]
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/954732/0d11bc05-5769-42ce-ba58-021ff0f323f8)

**Reply by Larockus** [reply]
Looking at the logs I've found the following. Which appears that the configuration changes for the Meraki have successfully been applied to the agent. but I'm still not seeing any events being forwarded from the appliance.

**Reply by Larockus** [reply]
![Meraki](https://github.com/Security-Onion-Solutions/securityonion/assets/108535113/29c88638-4ac4-4643-b433-79846de9bc74)

**Reply by dougburks** [reply]
Is the Meraki appliance supposed to send the logs to the Elastic Agent?

Are there any firewalls or other network devices that may be interfering with the traffic?

Have you tried using tcpdump to verify that the traffic is arriving?
